public/event-data.json

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
We don't need to push this file as it's from BuildGT and is out of date – instead, can we make a file called `event-data.template.json` with two generic sample events so people adding the schedule in the future know what to write to the JSON